Frameworks/AddressBook/ABAddressBookManagerInternal.h

```diff
@@ -17,10 +17,25 @@
 #pragma once
 
 #import "UWP/WindowsApplicationModelContacts.h"
+#import <AddressBook/ABRecord.h>
 
 @interface _ABAddressBookManager : NSObject
```

Comment:
this is not a good name for a class. what does a 'manager' do, in particular?

(consider just changing it to _ABAddressBook? it seems to serve a similar purpose as your previous _ classes, and the 'manager' bit adds unnecessary ambiguity, unless there's some exception to the pattern that I'm not seeing)